AWS/instances.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+
+function usage {
+    echo "usage: ./instances.sh [action] [config-file]"
+    echo "  	 [action] start,stop,state"
+    echo "  	 [config] valid route to a config file"
```

Comment:
Add here a real example of how to use it e.g.  `./instances.sh start file.config`